### PR TITLE
set asmt name when creating stub assessments

### PR DESCRIPTION
**Pull request by malaffoon**
Yeah, i know this is temporary and all.

### CODE REVIEW

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/ExamProcessorConfiguration.java [hunk 1]
```diff
@@ -39,15 +39,19 @@ public void process(final Message<?> message) {
         final byte[] payload = (byte[]) message.getPayload();
         final long importId = accessor.getImportId();
 
-        logger.info("received {} import {}", accessor.getContent(), importId);
+        logger.debug("received {} import {}", accessor.getContent(), importId);
```
Comment (malaffoon) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/ExamProcessorConfiguration.java]: unrelated: tweak log messages and update import record for a couple of code paths ...

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentService.java [hunk 2]
```diff
@@ -38,13 +38,14 @@ public Assessment findOneByNaturalId(final TDSReport tdsReport) {
         final boolean iab = test.getTestId().contains("IAB");
         final Map<String, Integer> items = newHashMap();
         for (final Opportunity.Item item : tdsReport.getOpportunity().getItem()) {
-            final String naturalId = new Long(item.getBankKey()).toString() + "-" + item.getKey();
+            final String naturalId = Long.toString(item.getBankKey()) + "-" + item.getKey();
```
Comment (malaffoon) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentService.java]: unrelated, IntelliJ was bugging me to change this.

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentService.java [hunk 3]
```diff
@@ -38,13 +38,14 @@ public Assessment findOneByNaturalId(final TDSReport tdsReport) {
...
             items.put(naturalId, null);
         }
         return stubbedOutAssessmentRepository.create(Assessment.builder()
+                .naturalId(test.getName())
+                .name(test.getTestId())
```
Comment (agorina) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentService.java]: This is so fun to read :)

**Comment by agorina** [comment]
I realized that we need a label for the UI. I believe this is what we are supposed to use. @jtreuting , could you please confirm?

@esotrope, just FYI.

**Pull request merged**